gluoncv/model_zoo/resnet.py

```diff
@@ -106,18 +110,18 @@ def hybrid_forward(self, F, x):
         x = self.body(x)
 
         if self.se:
-            w = F.contrib.AdaptiveAvgPooling2D(x, output_size=1)
+            w = F.contrib.AdaptiveAvgPooling2D(x.as_nd_ndarray(), output_size=1).as_np_ndarray()
```

Comment:
this is fine for a temporary solution, but I am expecting the operator will be available in `npx`